frontends/main/src/app/styled.tsx

```diff
@@ -13,7 +13,7 @@ export const PageWrapper = styled.div(({ theme }) => ({
   flexDirection: "column",
   height: "calc(100vh - 72px)",
   marginTop: "72px",
-  [theme.breakpoints.down("sm")]: {
+  [theme.breakpoints.down("md")]: {
     marginTop: "60px",
     height: "calc(100vh - 60px)",
```

Comment:
Could you switch this and the 72px above to `theme.custom.dimensions.headerHeightSm`, `theme.custom.dimensions.headerHeight`. (Maybe smoot should have a `theme.custom.dimensions.headerHeightMd`.)

I'd also do a find-all for a few other places that need to be changed. One is the left-side nav bar.